### ISSUE TITLE
All embarked units have strength of 25?

### ISSUE THREAD

**Issue by jahodovykolac** [yairm210/Unciv#3786]
When I attack any embarked unit, it always shows (and calculates, too) it has strength of 25 instead of its normal strength. But naval units are ok. Is this intentional?

**Comment by yairm210** [comment]
Yes, as per Civ V, embarked units have a defensive strength dependant on the era.
You can think of it as the strength of the transporting ships.

**Comment by jahodovykolac** [comment]
Okay, thanks
